lib/structs/jira/jira_account.py

```diff
@@ -0,0 +1,52 @@
+from dataclasses import dataclass, fields
+from typing import Dict
+
+
+@dataclass
+class JiraAccount:
+    """
+    Jira account Parameters dictating config info dictating how to create an issue.
+    :param username: Atlassian username
+    :param api_token: Atlassian Token
+    :param atlassian_endpoint: Atlassian server name to use
+    """
+
+    username: str
+    api_token: str
+    atlassian_endpoint: str
+
+    @staticmethod
+    def from_dict(dictionary: Dict):
+        """
+        Returns instance of this dataclass from a dictionary (for loading from config)
+        """
+        field_set = {field.name for field in fields(JiraAccount) if field.init}
+        filtered_arg_dict = {
+            key: value for key, value in dictionary.items() if key in field_set
+        }
+        return JiraAccount(**filtered_arg_dict)
+
+    @staticmethod
+    def from_pack_config(pack_config: dict, jira_account_name: str):

```

Comment:
you are missing tests for this function and `from_dict` function - see  https://github.com/stfc/st2-cloud-pack/blob/main/tests/structs/test_smtp_account.py for similar tests